packages/stark-build/config/build-utils.js

```diff
@@ -73,14 +80,18 @@ function getNbbAssetsConfig() {
 	const nbbPackages = ngCliUtils.getDirectoriesNames(helpers.root(NBB_MODULES_PATH));
 
 	for (const nbbPackage of nbbPackages) {
-		const ngCliConfigPath = helpers.root(NBB_MODULES_PATH + "/" + nbbPackage + "/" + ".ng-cli-config.json");
+		const ngCliConfigPath = helpers.root(NBB_MODULES_PATH + "/" + nbbPackage + "/" + "angular.json");
 
 		if (fs.existsSync(ngCliConfigPath)) {
 			const packageCliConfig = require(ngCliConfigPath);
 			const cliConfig = ngCliUtils.validateAngularCLIConfig(packageCliConfig);
 			if (cliConfig) {
-				if (cliConfig["apps"] && cliConfig["apps"][0] && cliConfig["apps"][0].assets) {
-					customAssets = [...customAssets, ...getCopyWebpackPluginConfig(cliConfig["apps"][0].assets)];
+				let cliProjectConfig;
+				if (cliConfig["defaultProject"] && (cliProjectConfig = cliConfig["projects"][cliConfig["defaultProject"]])) {
```

Comment:
Could you add some comments here to describe what is this for? It is not clear at first glance.

Another thing I found confusing is assigning a variable inside an IF... that makes the code less readable in my opinion :(

Again, I think we should just use the dot syntax whenever possible...

Reply:
done